### ISSUE TITLE
Your baseline model do not converge

### ISSUE THREAD

**Issue by junleiz** [fxmeng/filter-grafting#6]
Is there any bugs in your baseline.py? I ran the resnet32 on cifar100, but the acc is only 4%

**Comment by junleiz** [comment]
I also tried the cifar10 baseline. The model does not converge either. I do not change the file.
Files already downloaded and verified
Files already downloaded and verified
epoch:0    accuracy:39.900    best:39.900epoch:1    accuracy:39.990    best:39.990
epoch:2    accuracy:39.800    best:39.990
epoch:3    accuracy:39.890    best:39.990
epoch:4    accuracy:39.850    best:39.990
epoch:5    accuracy:39.660    best:39.990
epoch:6    accuracy:39.820    best:39.990
epoch:7    accuracy:39.770    best:39.990
epoch:8    accuracy:39.850    best:39.990
epoch:9    accuracy:40.120    best:40.120
epoch:10    accuracy:39.660    best:40.120
epoch:11    accuracy:39.770    best:40.120
epoch:12    accuracy:39.960    best:40.120
epoch:13    accuracy:40.070    best:40.120
epoch:14    accuracy:39.820    best:40.120
epoch:15    accuracy:39.740    best:40.120
epoch:16    accuracy:39.800    best:40.120
epoch:17    accuracy:39.860    best:40.120
epoch:18    accuracy:39.670    best:40.120
epoch:19    accuracy:39.650    best:40.120
epoch:20    accuracy:39.600    best:40.120
epoch:21    accuracy:39.850    best:40.120
epoch:22    accuracy:39.780    best:40.120
epoch:23    accuracy:39.840    best:40.120
epoch:24    accuracy:40.000    best:40.120
epoch:25    accuracy:39.700    best:40.120
epoch:26    accuracy:39.940    best:40.120
epoch:27    accuracy:39.850    best:40.120
epoch:28    accuracy:39.780    best:40.120
epoch:29    accuracy:39.860    best:40.120
epoch:30    accuracy:39.560    best:40.120
epoch:31    accuracy:39.920    best:40.120
epoch:32    accuracy:39.930    best:40.120
epoch:33    accuracy:39.860    best:40.120
epoch:34    accuracy:39.750    best:40.120
epoch:35    accuracy:39.940    best:40.120
epoch:36    accuracy:39.870    best:40.120

**Comment by fxmeng** [comment]
Thank you for your reminder, 
I have checked the baseline program and found that there was a problem with the indentation when rearranging the code：
The last line of the training function - lr_scheduler.step (), should be indented forward. That is, one step is updated for each epoch. However, the previous step was updated for each batch, so the learning rate will be too small to converge. The corresponding changes have been updated in the code, welcome to try again.
